etc/build/make_changelog.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+if [ $# -ne 1 ]
+then
+    echo "Need to specify change log file used in goreleaser"
+    exit 1
+fi
+
+mkdir -p "$(dirname "$1")" && touch "$1"
+git diff HEAD^ HEAD -- CHANGELOG.md | sed -n /##/,/##/p | grep -v "##" | cut -d '+' -f 2 > "$1"
```

Comment:
**Note for future**: I just now noticed, after reading this, that all of the 1.V.x release notes get written into the 1.V branch (`CHANGELOG.md` in `master` stops at 1.11.0, because we write those updates into the 1.11.x branch). I just wanted to note here that whatever process we end up using for `make doc` to get `1.11.x` docs changes served from master, we might need to apply to `CHANGELOG.md` as well.